src/views/dashboard.js

```diff
@@ -377,12 +377,11 @@ export default function Dashboard() {
 
 const useStyles = makeStyles((theme) => ({
 	container: {
-		width: "max-content",
+		maxWidth: "max-content",
 		minWidth: "100vw",
```

Comment:
It's still has some space between the table and the left corner. I think change `minWidth` to be less then `100vh` can fixed the problem.